src/mesido/gas_physics_mixin.py

```diff
@@ -132,6 +132,10 @@ def __init__(self, *args, **kwargs):
 
         self._gas_pipe_topo_pipe_class_map = {}
 
+        self.__gas_pipe_disconnect_var = {}
+        self.__gas_pipe_disconnect_var_bounds = {}
```

Comment:
Only create the map, comment out the var and bounds

Reply:
done